kernel/rtlil.cc

```diff
@@ -47,6 +47,136 @@ int RTLIL::IdString::last_created_idx_ptr_;
 
 dict<std::string, std::string> RTLIL::constpad;
 
+const pool<IdString> RTLIL::builtin_ff_cell_types = {
```

Comment:
This might be a C++ Static Initialization Order Fiasco bug.

Reply:
Yeah, I've been worried about that. Any suggestions on a better place to do that?

(also, this is actually part of #1884, maybe let's review that first?)